painttiming.bs

```diff
@@ -197,21 +199,38 @@ Processing model {#sec-processing-model}
 Reporting paint timing {#sec-reporting-paint-timing}
 --------------------------------------------------------
 
+<h4 dfn>Previously reported paints</h4>
+Every [=Document=] has an associated [=set=] of previously reported paints, initiallized to an empty [=set=].
+
+<h4 dfn export>First Contentful Paint</h4>
+<div algorithm="Compute first contentful paint">
+    When asked to <dfn>compute first contentful paint</dfn> given |document|, perform the following steps:
```

Comment:
nit: this should probably be named `should report first contentful paint`. So overall this could say something like:
`To know whether a Document document should report first contentful paint, perform the following steps:`